Formula/luaradio.rb

```diff
@@ -0,0 +1,47 @@
+class Luaradio < Formula
+  desc "lightweight, embeddable flow graph signal processing framework for SDR"
+  homepage "http://luaradio.io/"
+  url "https://github.com/vsergeev/luaradio/archive/v0.1.0.tar.gz"
+  sha256 "4423f94c0bb259952eae7db11e4db4509dfa7c6e205798c35a095050c530212b"
```

Comment:
To allow easily building the latest version (with brew install luaradio --HEAD), can you add:

    head "https://github.com/vsergeev/luaradio.git"